### PR TITLE
Delete Offline Projects Fix

### LINKED ISSUE #338: Deleting Offline Project
**Problem**
There is no way to delete an offline project either the user has to clear the complete local storage which will delete all his offline projects.

**Describe the solution you'd like**
Adding Delete Button on the Project > Open Oflline.

open offline Project it will display a cross button on the right of the title.
when we click the on the button it will ask for confirmation from user. if the user confirms it will delete the project.

**Prototype**
look at the cross button.
proto

**Are you working on this? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by MrBartusek**
Fixes #338 
There is an old feature request that has been implemented but, the issue wasn't closed and it is not working now. I made it look better and work + this pr will finally close this issue

image

### CODE REVIEW

File: public/js/data.js [hunk 1]
```diff
@@ -777,7 +777,7 @@ deleteOfflineProject = function (id) {
         localStorage.setItem("projectList", JSON.stringify(projectList));
         $('#openProjectDialog').empty();
         for (idd in projectList) {
```
Comment (JoshHeng) [inline public/js/data.js]: Use `id` instead of `idd`

File: public/css/UX.css [hunk 2]
```diff
@@ -1,5 +1,4 @@
-.deleteOff {
-    color: red;
+.deleteOfflineProject {
```
Comment (JoshHeng) [inline public/css/UX.css]: Maybe move to .scss files in `app/assets/stylesheets`

Comment (MrBartusek) [reply public/css/UX.css]: that can be done when stylesheets of a simulator will be cleaned not a task for this pr I think

**Comment by satu0king** [comment]
Good Job

**Pull request merged**